deploy/helm_charts/envs/mixer_prod.yaml

```diff
@@ -11,16 +11,16 @@ ingress:
     }
 
 serviceGroups:
-  - mixer-svg-service:
+  - svg:
       urlPaths:
         - "/place/stat-var-group/*"
         - "/stat-var/*"
         - "/v1/info/variable/group/*"
         - "/v1/variable/*"
       replicas:
-        default: 30
-        min: 30
-        max: 50
+        default: 10
```

Comment:
These API is very low cost in memory key value read and should be serve high qps per pod, hence reducing the replicas.